docs/user/ppl/cmd/fillnull.rst

```diff
@@ -0,0 +1,67 @@
+=============
+fillnull
+=============
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Description
+============
+| The ``fillnull`` command replaces null values for one or more fields.
+
+
+Syntax
+============
+fillnull "with" <expression> <field> ["," <field> ]...
+
+* field: mandatory. Name of an existing field that was piped into ``fillnull``. Null values for all specified fields are replaced with the value of expression.
+* expression: mandatory. Any expression support by the system. The expression value type must match the type of field.
+
+fillnull "using" <field> "=" <expression> ["," <field> "=" <expression> ]...
+
+* field: mandatory. Name of an existing field that was piped into ``fillnull``.
+* expression: mandatory. Any expression support by the system. The expression value type must match the type of field.
+
+Example 1: Replace null values with the same value for multiple fields
+======================================================================
+
+The example show to replace null values for email and host with "<not found>".
+
+PPL query::
+
+    os> source=accounts | fields email, host | fillnull with '<not found>' email, host ;
+    fetched rows / total rows = 4/4
+    +-----------------------+------------+
+    | email                 | host       |
+    |-----------------------+------------|
+    | amberduke@pyrami.com  | pyrami.com |
+    | hattiebond@netagy.com | netagy.com |
+    | <not found>           |            |
+    | daleadams@boink.com   | boink.com  |
+    +-----------------------+------------+
+
+Example 2: Replace null values for multiple fields with different values
+========================================================================
+
+The example show to replace null values for email with "<not found>" and null values for host with "<no host>".
+
+PPL query::
+
+    os> source=accounts | fields email, host | fillnull using email = '<not found>', host = '<no host>' ;
+    fetched rows / total rows = 4/4
+    +-----------------------+------------+
+    | email                 | host       |
+    |-----------------------+------------|
+    | amberduke@pyrami.com  | pyrami.com |
+    | hattiebond@netagy.com | netagy.com |
+    | <not found>           |            |
+    | daleadams@boink.com   | boink.com  |
+    +-----------------------+------------+
```

Comment:
This is missing `<no host>` from the right column, right? Same for the table above, it looks like one `<not found>` is missing?

```suggestion
    os> source=accounts | fields email, host | fillnull using email = '<not found>', host = '<no host>' ;
    fetched rows / total rows = 4/4
    +-----------------------+------------+
    | email                 | host       |
    |-----------------------+------------|
    | amberduke@pyrami.com  | pyrami.com |
    | hattiebond@netagy.com | netagy.com |
    | <not found>           | <no host>  |
    | daleadams@boink.com   | boink.com  |
    +-----------------------+------------+
```

Reply:
Updated the test.

Reply:
In Example 1, it looks like `<not found>` is still missing from the `host` column..?

Reply:
The test has been changed to use `employer` rather than `host`. `null` values are replaced with `<no employer>`.

Reply:
Looks good to me. Thanks.